tests/src/performance/linkbench/scripts/build.cmd

```diff
@@ -132,11 +132,10 @@ cd ..
 
 REM Build CscCore
 call Restore.cmd
-cd src\Compilers\CSharp\CscCore
-call %__dotnet2% publish -c Release -r win7-x64
+for /f "delims=`" %%i in ('powershell -noprofile -executionPolicy RemoteSigned -command "& { . build\scripts\build-utils.ps1;Ensure-MSBuild }"') do set MSBUILD=%%i
```

Comment:
`Ensure-MsBuild` finds the appropriate version of MSBuild to be used in the next step.